doc/manual/building-from-source.rst

```diff
@@ -184,6 +162,44 @@ and run this after any changes::
     sudo ldconfig
 
 
+.. _dependencies_on_macos:
+
+macOS
+^^^^^
+
+We recommend installing all dependencies from Homebrew_::
+
+    brew install make automake libtool pkg-config help2man fftw asio fmt vrpn freeglut yarn SoundScapeRenderer/ssr/libmysofa llvm
```

Comment:
Is `freeglut` actually needed?

And `yarn` is only needed for the browser GUI and only when building from the Git repo.
Otherwise this can probably be skipped, since it might also be a large dependency?

OTOH, `ecasound`, `jack`, `libsndfile` and `websocketpp` should probably be added to the list?
And what about `qt@5`?